findbugs-exclude-filter.xml

```diff
@@ -0,0 +1,104 @@
+            <Bug pattern="HARD_CODE_PASSWORD" />
+        </Match>
+    </FindBugsFilter>
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.api.server.keystore.management.v1.core.KeyStoreService" />
+            <Bug pattern="PATH_TRAVERSAL_IN" />
+        </Match>
+    </FindBugsFilter>
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.rest.api.server.challenge.v1.core.ServerChallengeService" />
+            <Method name="patchChallengeSet" />
+            <Bug pattern="IMPROPER_UNICODE" />
+        </Match>
+    </FindBugsFilter>
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.api.server.application.management.v1.core.functions.application.ServiceProviderToApiModel" />
+            <Method name="isResidentSp" />
+            <Bug pattern="IMPROPER_UNICODE" />
+        </Match>
+    </FindBugsFilter>
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.api.server.application.management.v1.core.functions.application.UpdateAuthenticationSequence" />
+            <Method name="isRevertToDefaultSequence" />
+            <Bug pattern="IMPROPER_UNICODE" />
+        </Match>
+    </FindBugsFilter>
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.api.server.tenant.management.v1.core.ServerTenantManagementService" />
+            <Method name="createTenantInfoBean" />
+            <Bug pattern="IMPROPER_UNICODE" />
+        </Match>
+    </FindBugsFilter>
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.api.server.tenant.management.v1.core.ServerTenantManagementService" />
+            <Method name="validateInputAgainstCode" />
+            <Bug pattern="IMPROPER_UNICODE" />
+        </Match>
+    </FindBugsFilter>
+
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.rest.api.server.claim.management.v1.core.ServerClaimManagementService" />
+            <Method name="validateAttributeMappings" />
+            <Bug pattern="IMPROPER_UNICODE" />
+        </Match>
+    </FindBugsFilter>
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.api.server.application.management.v1.core.functions.Utils" />
+            <Method name="deepCopyApplication" />
+            <Bug pattern="OBJECT_DESERIALIZATION" />
+        </Match>
+    </FindBugsFilter>
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.api.server.script.library.v1.core.ServerScriptLibrariesService" />
+            <Or>
+                <Method name="addScriptLibrary" />
+                <Method name="updateScriptLibrary" />
+            </Or>
+            <Bug pattern="PATH_TRAVERSAL_IN" />
+        </Match>
+    </FindBugsFilter>
+</FindBugsFilter>
```

Comment:
Add new line at the end of the file

Reply:
fixed in 133c6682327014072c3c19ad62360620a9904b91